src/redux/accountSlice.js

```diff
@@ -0,0 +1,16 @@
+import {createSlice} from '@reduxjs/toolkit'
+
+const accountSlice = createSlice({
+    name:'accounts',
+    initialState:{
+        accounts:[]
+    },
+    reducers:{
+        getAccounts(state,action){
+            state.accounts = action.payload; 
+        }
+    }
+}); 
+
+export const {getAccounts} = accountSlice.actions;
+export default accountSlice.reducer; 
```

Comment:
I'd rename this file to `accountReducer.js` as _slice_ - even though it comes from the Redux Toolkit package - isn't too intuitive at first.